src/patternfly/components/Tabs/tabs.scss

```diff
@@ -97,6 +97,8 @@
   overflow-x: auto;
   -webkit-overflow-scrolling: touch;
 
+  @include pf-overflow-hide-scroll;
```

Comment:
one more thing, can you move this to the top of the selector? `@include` should come after variables but before properties.

Reply:
thanks @mcoker updated